src/transformers/tokenization_utils_base.py

```diff
@@ -2279,6 +2279,12 @@ def __call__(
         )
 
         if is_batched:
+            if isinstance(text_pair, str):
+                text_pair = [text_pair] * len(text)
```

Comment:
This is very fastai-like :-) In Transformers, the general philosophy is to avoid any magic like that and let the user explicitly do the conversion. In 99% of the cases, having one text for `text_pair` and a list for `text` is an actual mistake of the user, and for the remaining 1%, the user can pass that list `[text_pair] * len(text)` themselves.

So I would just raise an error here!

Reply:
Thanks for being a bridge for me from fastai!  Haha yes this is exactly where I learned this!  I added an error message instead 🙇🏽